tembo-operator/src/extensions/database_queries.rs

```diff
@@ -33,7 +33,7 @@ pub fn check_input(input: &str) -> bool {
 pub const LIST_SHARED_PRELOAD_LIBRARIES_QUERY: &str = r#"SHOW shared_preload_libraries;"#;
 
 pub const LIST_DATABASES_QUERY: &str =
-    r#"SELECT datname FROM pg_database WHERE datistemplate = false;"#;
+    r#"SELECT datname FROM pg_database WHERE datname != 'template0';"#;
```

Comment:
only exclude `template0`. That database is not configured to accept any connections. `template1` does allow it though, and `template1` is where we will install Stack extensions into.